### DISCUSSION TITLE
/io/buffer.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#60] [Announcements]
# /io/buffer.html

Java程序员进阶之路，小白的零基础Java教程，Java缓冲流（Buffered）：读写速度有了质的飞升

https://tobebetterjavaer.com/io/buffer.html

**Comment by Rain2Cloud** [comment]
二哥，最后的字符缓冲流练习**代码实现**中，我在idea里面debug运行看到，在把**好诗**保存到Map里面的时候，当while循环完毕时，Map里面的数据已经自动按**好诗的编号**给排好序了，这是为啥zei？插入的时候是乱序的，而且Map的key不就是无序的么？
